src/interface/src/app/shared/_variables.scss

```diff
@@ -1,3 +1,28 @@
 $toolbar-height: 48px;
 $navbar-height: 56px;
 $info-bar-height: 50px;
+
+// Planscape Style Guide colors
```

Comment:
we already have a `_colors` file with this, so this is repeated

Reply:
oops, yeah, thanks. This was definitely old.